ngx-fudis/projects/ngx-fudis/src/lib/components/form/date/datepicker/datepicker.stories.ts

```diff
@@ -217,3 +198,21 @@ WithMinMaxValidator.args = {
   ...commonArgs,
   helpText: 'Choose a date between the allowed range.',
 };
+
+export const WithDateFilter: StoryFn = (args) => ({
+  props: {
+    ...args,
+    addError: action('addError'),
+    removeError: action('removeError'),
+  },
+  template: `
```

Comment:
Add `html` tag before the back-ticked content (like all other examples in this file).